Sprint-3/revise/implement/repeat.test.js

```diff
@@ -23,3 +23,29 @@
 // Given a target string str and a negative integer count,
 // When the repeat function is called with these inputs,
 // Then it should throw an error or return an appropriate error message, as negative counts are not valid.
+
+function repeat(str, count) {
+  if (count < 0) {
+    return 'Count cannot be negative';
+  }
+  return str.repeat(count);
```

Comment:
While the solution is correct, I think you're expected to create your own function instead of relying on an already existing method that does exactly that